CreeDictionary/CreeDictionary/settings.py

```diff
@@ -150,6 +150,25 @@
     {"NAME": "django.contrib.auth.password_validation.NumericPasswordValidator"},
 ]
 
+############################### Morphodict configuration ###############################
+
+# Configure the orthographies
+MORPHODICT_ORTHOGRAPHY = {
+    # 'Latn' is Okimāsis/Wolvegrey's SRO
+    "default": "Latn",
+    "available": {
+        "Latn": {"name": "SRO (êîôâ)"},
+        "Latn-x-macron": {
+            "name": "SRO (ēīōā)",
+            "converter": "CreeDictionary.orthography.to_macrons",
+        },
+        "Cans": {
+            "name": "Syllabics",
+            "converter": "CreeDictionary.orthography.to_syllabics",
+        },
+    },
+}
```

Comment:
This is how orthographies in languages are configured. Note: given our talks with MESC, I think I might end up adding some more orthographies to this list for Plains Cree...